chart/githubconnector/templates/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.name }}
+    chart: {{ include "github-connector-chart.chart" . }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Values.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.name }}
+    spec:
+      containers:
+      - name: {{ .Values.container.name }}
```

Comment:
No need to extract container name to values. It has to be unique in Pod scope so you may as well hardcode it. And make it short, you will have to type it when getting logs ;). If there is only one container I suggest something like `main`.

Reply:
For now we often change used image for tests and it's easier to change it during installation than hardcode it, but for future it's definitely better idea